server/routes/profile.js

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+
+import { createProfile, updateProfile, getAllProfiles, getProfile } from '../controllers/profile.js';
+const protect = require("../middleware/auth");
+
+const router = express.Router();
```

Comment:
this looks good! I noticed that while the routers were attached here, how would these profile routes be accessed? because I don't see these attached to the `app` 

would you be able to provide us the exact route to test these?